Command/EncryptPasswordCommand.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the ekino/data-protection-bundle project.
+ *
+ * (c) Ekino
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Ekino\DataProtectionBundle\Command;
+
+use Ekino\DataProtectionBundle\DependencyInjection\Configuration;
+use Ekino\DataProtectionBundle\Encryptor\Encryptor;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class EncryptPasswordCommand
+ *
+ * @author Christian Kollross <christian.kollross@ekino.com>
+ */
+final class EncryptPasswordCommand extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected static $defaultName = 'ekino-data-protection:encrypt';
+
+    /**
+     * @var string
+     */
+    private $secret;
+
+    public function __construct(string $secret)
+    {
+        parent::__construct();
+
+        $this->secret = $secret;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setDescription('Encrypt a short text (like a password) with a given secret')
+            ->addArgument('text', null, 'Text to encrypt')
+            ->addOption('secret', 's', InputOption::VALUE_REQUIRED, 'Application secret')
+            ->addOption('method', 'm', InputOption::VALUE_REQUIRED, 'Encryption cypher, see openssl_get_cipher_methods()',
+                Configuration::DEFAULT_METHOD)
+            ->setHelp('Usage: `bin/console ekino-data-protection:encrypt myText`, optionally with `--secret mySecret` and/or `--method myCypher`')
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $text   = $input->getArgument('text');
+        $secret = $input->getOption('secret') ?: $this->secret;
+
+        if (empty($text) || empty($secret)) {
```

Comment:
`$text` cannot be empty because it's an argument, so Symfony will raise an error before.